src/java/org/jivesoftware/openfire/roster/RosterManager.java

```diff
@@ -77,6 +77,15 @@ public static boolean isRosterServiceEnabled() {
         return JiveGlobals.getBooleanProperty("xmpp.client.roster.active", true);
     }
 
+    /**
+     * Returns true if the roster versioning is enabled.
+     *
+     * @return true if the roster versioning is enabled.
+     */
+    public static boolean isRosterVersioningEnabled() {
+        return JiveGlobals.getBooleanProperty("xmpp.client.roster.versioning.active", false);
```

Comment:
Why not have a default value of `true`? It'll only be used by clients that understand the protocol anyways, right?